tests/tools/_perturbation_space/test_simple_cluster_space.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+import pertpy as pt
+
+
+def test_clustering():
+    X = np.random.rand(10, 5)
+    obs = pd.DataFrame(
+        {
+            "perturbations": [
+                "control",
+                "target1",
+                "target1",
+                "target2",
+                "target2",
+                "target1",
+                "target1",
+                "target2",
+                "target2",
+                "target2",
+            ]
+        }
+    )
+    adata = AnnData(X, obs=obs)
+
+    # Compute clustering at observation level
```

Comment:
Don't just compute - test for the expected result!

Reply:
done